tests/juniper_junos/show_system_processes_brief/juniper_junos_show_system_processes_brief.yml

```
@@ -0,0 +1,35 @@
+---
+parsed_sample:
+  - memory_active: "31554.4"
+    memory_buffer: "3661.7"
+    memory_free: "17292.4"
+    memory_used: "10600.3"
+    memoryory_format: "M"
+    node: "fpc0"
+    swap_avail: "19854.3"
+    swap_format: "M"
+    swap_free: "100.0"
+    swap_total: "100.0"
+    swap_used: "0.0"
+  - memory_active: "128191.6"
+    memory_buffer: "45096.8"
+    memory_free: "65543.4"
+    memory_used: "17551.4"
+    memoryory_format: "M"
+    node: "re0"
+    swap_avail: "108490.7"
+    swap_format: "M"
+    swap_free: "8192.0"
+    swap_total: "8192.0"
+    swap_used: "0.0"
+  - memory_active: "128191.6"
+    memory_buffer: "28674.5"
+    memory_free: "93230.9"
+    memory_used: "6286.2"
+    memoryory_format: "M"
```

Resolve typo by removing extra "ory" from capture group name.

```suggestion
    memory_format: "M"
    node: "fpc0"
    swap_avail: "19854.3"
    swap_format: "M"
    swap_free: "100.0"
    swap_total: "100.0"
    swap_used: "0.0"
  - memory_active: "128191.6"
    memory_buffer: "45096.8"
    memory_free: "65543.4"
    memory_used: "17551.4"
    memory_format: "M"
    node: "re0"
    swap_avail: "108490.7"
    swap_format: "M"
    swap_free: "8192.0"
    swap_total: "8192.0"
    swap_used: "0.0"
  - memory_active: "128191.6"
    memory_buffer: "28674.5"
    memory_free: "93230.9"
    memory_used: "6286.2"
    memory_format: "M"
```

---

I did a mass find and replace when I updated the memory variables. *(&% I caught this in the template itself, but not the resulting output file.